src/blueprints/timeline-state-resolver-types/viz-types.ts

```diff
@@ -43,20 +43,23 @@ export interface VizMseContinueTimelineObject extends TimelineObject {
   content: {
     deviceType: DeviceType.VIZMSE
     type: VizType.CONTINUE
-    direction: -1 | 1 | undefined
+    direction: VizMseContinueDirection
```

Comment:
It appears to no longer be optional. 
I take that we currently, don't have anywhere that matters?

Reply:
When would you create a `continue` object and not tell which direction it should go?

We currently only create one `continue` TimelineObject and we always set it to "forward"